include/openssl/bio.h

```diff
@@ -396,10 +396,15 @@ OPENSSL_EXPORT int BIO_read_asn1(BIO *bio, uint8_t **out, size_t *out_len,
 //
 // |BIO_ctrl_pending| returns the number of bytes currently stored.
 
-// BIO_NOCLOSE and |BIO_CLOSE| can be used as symbolic arguments when a "close
-// flag" is passed to a BIO function.
+// |BIO_NOCLOSE|, |BIO_CLOSE|, and |BIO_FP_TEXT|  can be used as symbolic
+// arguments when a "close flag" is passed to a BIO function. |BIO_CLOSE| (on
+// by default) will free the backing buffer on BIO close, whereas |BIO_NOCLOSE|
+// will not. Setting |BIO_FP_TEXT| will cause the file to be opened as a text
+// file (this is only relevant on Windows due to CRLF endings) instead of the
+// default behavior of opening the file in binary mode.
 #define BIO_NOCLOSE 0
 #define BIO_CLOSE 1
+#define BIO_FP_TEXT 0x10
```

Comment:
If you wanted to you could create a new section (2 blank lines, title and description followed by 1 blank lines) and then document each one:
```suggestion

// BIO close flags
// These  can be used as symbolic arguments when a "close flag" is passed to a BIO function. 

// BIO_NOCLOSE does foo
#define BIO_NOCLOSE 0
// BIO_CLOSE does bar
#define BIO_CLOSE 1
// BIO_FP_TEXT does bazz
#define BIO_FP_TEXT 0x10
```
See some of the bn.h examples https://github.com/aws/aws-lc/blob/main/include/openssl/bn.h#L866-L869

Reply:
good call, will do.